winui/Rating/Overview.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
```

Comment:
The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service, such as movie applications, products, and more.

Reply:
Changed the content as mentioned above.